src/postgresql.c

```diff
@@ -742,7 +742,7 @@ static char *values_to_sqlarray(const data_set_t *ds, const value_list_t *vl,
 
       status = snprintf(str_ptr, str_len, ",%lf", rates[i]);
     } else if (ds->ds[i].type == DS_TYPE_COUNTER)
-      status = snprintf(str_ptr, str_len, ",%llu", vl->values[i].counter);
+      status = snprintf(str_ptr, str_len, ",%" PRIu64, vl->values[i].counter);
```

Comment:
Do you need a cast here?

Reply:
I think you're right with all of these. I was able to build and install without error, but it probably wasn't building these plugins and so didn't throw any error.